Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_Activity_Behavior_Hunting_playbook.yml

```diff
@@ -1,7 +1,7 @@
 id: SolarStorm Activity Behavior Hunting playbook
 version: -1
 name: SolarStorm Activity Behavior Hunting playbook
-description: "This manual playbook helps you to discover SolarStorm activity behavior.\
+description: "This manual playbook is triggered by a 'SolarStorm and SUNBURST Hunting and Response Playbook' playbook and helps you to discover SolarStorm activity behavior.\
```

Comment:
description: "This manual playbook should be used as a sublaybook in the 'SolarStorm and SUNBURST Hunting and Response' playbook and it helps you hunt for suspicious behavior related to SolarStorm activity. \